sycl/test-e2e/Basic/built-ins/marray_geometric.cpp

```diff
@@ -1,5 +1,6 @@
 // RUN: %{build} -o %t.out
 // RUN: %{run} %t.out
+// XFAIL: cpu
```

Comment:
Could you open a Github issue for the XFAIL'd tests? And add a link to that here. That way, it would be easier for other developers to understand why these tests were XFAIL'd.

Reply:
Thanks for the suggestion, open https://github.com/intel/llvm/issues/13397 for the new marked 'XFAIL' tests.